protocol/protobuf/community_shard_key.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+option go_package = "./;protobuf";
+
+package protobuf;
+
+message CommunityShardKey {
+  bytes communityID = 1;
+  bytes privateKey = 2;
+  string topic = 3;
+  uint64 clock = 4;
+  int32 shard_cluster = 5;
```

Comment:
Having a quick look, it seems we're not being completely consistent with the usage of snake_case across .proto files. Since it seems that convention is used for the majority of protobuffers of status-go, i'll change this file to also use snake_case